client/app/helper/mpc.js

```diff
@@ -48,7 +48,8 @@ define(['constants'], function (constants) {
     var questions = [];
     var usability = [];
     var table_meta = {};
-
+    const threshold=table_template.cohort_threshold
```

Comment:
Just being nitpicky, but should use consistent spacing e.g. `threshold = table...`
But given the existing quality of the codebase,...mmmm might if we end up doing a complete re-write.
But my same comment about spacing applies to the rest of this PR.

Reply:
Yeah agreed. We might eventually end up rewriting whole but it's worth applying consistent spacing for this PR.Thank you Justin.